src/Microsoft.AspNet.Server.Kestrel/Networking/SockAddr.cs

```diff
@@ -41,18 +41,37 @@ public IPEndPoint GetIPEndPoint()
             // 0000 0000 0000 0000
             //
             // Example 3: 10.135.34.141:12804, not dual-stack sockets
+            //
             // 8d22 870a fd31 0002  => sa_family == AF_INET (02)
             // 0000 0000 0000 0000
             // 0000 0000 0000 0000
             // 0000 0000 0000 0000
-
+            //
+            // Example 4: 127.0.0.1:52798, on a Mac OS
+            //
+            // 0100 007F 3ECE 0210  => sa_family == AF_INET (02) Note that struct sockaddr on mac use
+            // 0000 0000 0000 0000     the second unint8 field for sa family type
+            // 0000 0000 0000 0000     http://www.opensource.apple.com/source/xnu/xnu-1456.1.26/bsd/sys/socket.h
+            // 0000 0000 0000 0000
+            //
+            // Reference:
+            //  - Windows: https://msdn.microsoft.com/en-us/library/windows/desktop/ms740506(v=vs.85).aspx
+            //  - Linux: https://github.com/torvalds/linux/blob/master/include/linux/socket.h
```

Comment:
linking to master is not future proof, better to use a permalink like https://github.com/torvalds/linux/blob/6a13feb9c82803e2b815eca72fa7a9f5561d7861/include/linux/socket.h